docs/whats-new.asciidoc

```diff
@@ -88,7 +89,7 @@ image::whats-new/images/8.11/cases-add-custom-field.png[Add custom fields to cas
 [float]
 === Connectors page renamed
 
-The page where you create and manage case connectors has been renamed to Settings.
+The page where you create and manage case connectors has been renamed to **Settings**.
```

Comment:
@jmikell821 just want to double-check if we should use bold for page names in this context? Per our guidelines and topics such as [Hosts page](https://www.elastic.co/guide/en/security/current/hosts-overview.html), we don't seem to use bold for pages unless we're referring to a clickable UI item (i.e. "Click **Cases** -> **Settings**").

Reply:
@natasha-moore-elastic understood about the guidelines, but I think we should bold it in this context for emphasis.